src/components/Layout/Card/creator-card.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import Image from "next/image";
+import type { propsType } from "lib/types";
+import { FC } from "react";
+
+export const CreatorCard: FC<propsType> = ({name, photo, link})=>{
```

Comment:
Name propsType is generic and confusing should me creatorCardProps

You can have deferent types in that folder they don't share same props